sdk/eventhub/tests.data.yml

```diff
@@ -6,5 +6,5 @@ extends:
     MaxParallel: 6
     ServiceDirectory: eventhub
     SDKType: data
-    TimeoutInMinutes: 190
-    Clouds: 'Public,Canary'
\ No newline at end of file
+    TimeoutInMinutes: 240
+    Clouds: 'Public,Preview,UsGov,China,Canary' 
```

Comment:
Do we want the track on tests running in sovereign clouds?    @serkantkaraca  should make the call.